test/contact_properties.js

```diff
@@ -1,164 +1,296 @@
-const chai = require('chai')
-const expect = chai.expect
-
+const { expect } = require('chai')
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: process.env.API_KEY || 'demo' })
-
-const property = {
-  name: 'mk_customer_fit_segment',
-  label: 'MadKudu Customer Fit',
-  groupName: 'contactinformation',
-  description: 'MadKudu Customer Fit',
-  type: 'string',
-  fieldType: 'text',
-  formField: false,
-  displayOrder: -1,
-  options: [],
-}
+const fakeHubspotApi = require('./helpers/fake_hubspot_api')
 
 describe('contacts.properties', function() {
+  const hubspot = new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'fake-token',
+  })
+  const properties = {
```

Comment:
Are these properties being used to create a Contact Property Group and a Contact Property? That seems a bit odd to me as they're not composed of the same fields. Fetching a Contact Property Group returns:
```
{
    "name": "colors2",
    "displayName": "Details about color choices",
    "displayOrder": 81690,
    "hubspotDefined": false
}
```

Fetching a Contact Property returns:
```
{
    "name": "account_type",
    "label": "account_type",
    "description": "",
    "groupName": "contactinformation",
    "type": "enumeration",
    "fieldType": "select",
    "options": []
}
```